network-api/networkapi/utility/middleware.py

```diff
@@ -2,6 +2,10 @@
 from django.conf import settings
 
 hostnames = settings.TARGET_DOMAINS
+
+if len(hostnames) == 0:
```

Comment:
we're checking `len()` here because settings.py loades in the target domains as `TARGET_DOMAINS=(list, []),` defaulting to an empty list if no env. var of the same name is found.